avutil/avutil.go

```diff
@@ -105,6 +116,24 @@ const (
 	PictureTypeBI   PictureType = C.AV_PICTURE_TYPE_BI
 )
 
+type FrameSideDataType C.enum_AVFrameSideDataType
+
+const (
+	FrameDataPanScan                  FrameSideDataType = C.AV_FRAME_DATA_PANSCAN
+	FrameDataA53CC                    FrameSideDataType = C.AV_FRAME_DATA_A53_CC
+	FrameDataStereo3D                 FrameSideDataType = C.AV_FRAME_DATA_STEREO3D
+	FrameDataMatrixEncoding           FrameSideDataType = C.AV_FRAME_DATA_MATRIXENCODING
```

Comment:
Do you mean we use `FrameDataMATRIX_ENCODING` instead of `FrameDataMatrixEncoding`?

Should I change all these types using `_`?

Thanks.